packages/openapi/src/openapi-request-builder.spec.ts

```diff
@@ -327,5 +328,13 @@ describe('openapi-request-builder', () => {
       const requestConfig = await requestBuilder['requestConfig']();
       expect(requestConfig['method']).toBe('merge');
     });
+    it('test the exposed type from SDK', async () => {
```

Comment:
What exactly is this supposed to test?
If you were trying to test that you can not use the omitted options in the narrowed type of `CustomRequestConfig`, I do not think you can properly test this, as the omitted options are still usable, due to `HttpRequstConfigBase`'s type of `key: string]: any;`

Reply:
Yeah I understand that the omitted options are still usable; the test was not intended for validation, but rather just to demonstrate the usability of the exposed type as users can now define custom configs using `CustomRequestConfig` type and pass them into `addCustomRequestConfiguration` method. Before this, only string values could be defined in the parameters for `addCustomRequestConfiguration`. I thought it would be worth adding this.

Reply:
I see, for that I believe proper documentation should be sufficient, we can remove the test. 👍